analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/ExplodedNodeTest.cs

```diff
@@ -131,5 +131,23 @@ public void GetHashCode_ReturnsSameForEquivalent()
             basic.GetHashCode().Should().NotBe(differentLocation.GetHashCode());
             basic.GetHashCode().Should().NotBe(differentState.GetHashCode());
         }
+
+        [TestMethod]
+        public void ToString_SerializeOperationAndState()
+        {
+            var cfg = TestHelper.CompileCfgBodyCS("var a = true;");
+            var state = ProgramState.Empty.SetSymbolValue(cfg.Blocks[1].Operations[0].Children.First().TrackedSymbol(), new(new()));
```

Comment:
`new(new())` -> hard to figure out what's happening here without looking at the method signature.

Reply:
That's a kind of build-in for target typed new feature. As we're inside UT, it is just a boring scaffolding.